xet-tools.rb

```diff
@@ -11,7 +11,7 @@ class XetTools < Formula
   end
 
   on_linux do
-    url "https://github.com/xetdata/xet-tools/releases/download/v0.7.1/xet-v0.7.1-linux-x86_64.tar.gz"
```

Comment:
Does this require changes to our circle-ci jobs for creating the file? What happens if we want to have a linux-arm build?